crates/matrix-sdk-ui/src/unable_to_decrypt_hook.rs

```diff
@@ -57,7 +57,7 @@ pub struct UnableToDecryptInfo {
     pub cause: UtdCause,
 }
 
-type PendingUtdReports = Vec<(OwnedEventId, JoinHandle<()>)>;
+type PendingUtdReports = HashMap<OwnedEventId, JoinHandle<()>>;
```

Comment:
mmm, ok. For me, the simplicity of the code was worth it. Trying to avoid the hash overhead smells of premature optimisation to me.

Not a strong opinion from me though; am happy to revert this.

Reply:
Yeah, let's keep it as you've changed it, it doesn't matter much to me either.